app/components/ui/Navigation.module.scss

```diff
@@ -172,3 +178,29 @@
     font-style: normal;
   }
 }
+
+.buttonLink {
+  appearance: button;
+  text-decoration: none;
+  color: white;
+  background-color: $color-brand;
+  border-radius: 3px;
+  font-size: 16px;
+  font-weight: 600;
+  height: 60px;
```

Comment:
I see that this matches the height in the Figma design, but I also noticed that in the Figma design, the header is now 100px tall instead of the 70px we have today, and this makes a huge difference in terms of the perceived size of the button relative to the whitespace. Could you increase the height of that header to match the Figma design?

Reply:
There is a $header-height constant in the our `_layoututils.scss` file that you can adjust this on.

Reply:
If we only want to change the header height when the button is present, will this constant be useful?

Reply:
If you're talking about the button not being present on most of the whitelabel sites, then I think it's probably fine to increase the header nav height on all the sites, even if there's no button, just for consistency. But this might be a question more for the design team.